src/components/Repositories.js

```diff
@@ -18,15 +18,17 @@ function Repositories({reposData, reposConfig}) {
     )
 }
 
-function Repository({name, description, imageUrl}) {
+function Repository({name, description, imageUrl, repoUrl}) {
     return (
         <div className={clsx('col col--4', styles.repository)}>
             <div className="text--center">
-                <img className={styles.repositoryImage} src={imageUrl} alt={name} />
+                <a href={repoUrl} target="_blank">
+                    <img className={styles.repositoryImage} src={imageUrl} alt={name} />
+                </a>
             </div>
             <div className={styles.repositoryTitle}>
                 <h3>{name}</h3>
```

Comment:
It's probably also worth making the name clickable as that's actually the first thing I tried. And then maybe the arrow isn't actually needed unless we think it looks nice design-wise? (I prefer it without but maybe others like it)

Reply:
I've added the link also to the repo name. As for the arrow, I also have a doubt, but on the other hand I see in the mock up that the same arrow is reused in multiple places, as to give a kind of consistency. Let's ask the opinion of our designer.

Reply:
As suggested by our designer, it's probably better - at least for now - to keep the arrow because it adds consistency and more explicitly highlights that there is a link to click. We can revisit this in the future.